packages/block-editor/src/components/inserter/media-tab/hooks.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useEffect, useState } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../../store';
+
+// TODO: Even though the `mature` param is `false` by default, we might need to determine where
+// and if there is other 'weird' content like in the title. It happened for me to test with `skate`
+// and the first result contains a word in the title that might not be suitable for all users.
```

Comment:
This is a difficult problem I can speak to a bit. There are two sources in Openverse that contain user-uploaded content:

- Flickr
- Wikimedia Commons

These are two of the largest sources in Openverse[^1], but both can contain 'mature' images that aren't correctly labeled as such. This goes beyond curse words, like you've identified in your example, and can potentially include explicit images of a sexual or violent nature. 

The fastest fix here I would reccommend, which is unfortunate because it excludes hundreds of millions of works, would be to exclude Flickr with the `?excluded_source=flickr`[^2] parameter. I think that would be a fine place to start while Openverse develops further methods for removing or hiding behind the mature flag these incorrectly categorized images.

Openverse is a search engine, not a gallery of curated content, so there are still possibilities that a result could be insensitive, in poor taste, or unexpected even after these changes. Or even more simply, low-quality! In light of this it's always possible that it might be useful to show some disclaimer on the page.

Finally, while probably too much for v1, we do have an [API endpoint for reporting media](https://api.openverse.engineering/v1/#operation/image_report) that would be wonderful to include, so users could help surface any results which are inappropriate or violate copyright, as examples.

[^1]: See https://wordpress.org/openverse/sources for a full list of sources and their media totals.
[^2]: For more info on this search parameter, see: https://api.openverse.engineering/v1/#operation/image_search

Reply:
> Finally, while probably too much for v1, we do have an [API endpoint for reporting media](https://api.openverse.engineering/v1/#operation/image_report) that would be wonderful to include, so users could help surface any results which are inappropriate or violate copyright, as examples.

This is a great idea, and even if we don't implement it initially I'll add a TODO item to design a report mechanism and a disclaimer message.

Reply:
That's great info Zack, thank you!